### ISSUE TITLE
python3.7 install numpy 1.14.2

### ISSUE THREAD

**Issue by ukos-git** [NeurodataWithoutBorders/pynwb#651]
## 1) Bug

Can not install requirements for python 3.7 using pip

### Steps to Reproduce
```
virtualenv --python=python3.7 env
pip install -r requirements.txt -r requirements-dev.txt
```
### Environment
Please describe your environment according to the following bullet points.
- **Python Executable:** python
- **Python Version:** Python 3.7.1rc1
- **Operating System:**Ubuntu 18.04
- **Pynwb Version:** 430ae547afb1f454c6e7df636914ea082fc36fc2

### Problem/Use Case
There seem to be CPython issues with [numpy 1.14.2](https://github.com/numpy/numpy/issues/10500#issuecomment-393949228). numpy and pandas work together.

Pandas package here is fairly old. Updating both, numpy, and pandas to recent versions fixes the problem. Note that current numpy 1.15 throws some warnings to the cli: #650 so 

I suggest an update of the following packages (as they depend on each other):

* numpy==1.14.2 --> numpy==1.14.6 (this is the last 1.14, although it should be fixed already in 1.14.3)
* pandas==0.19.2 --> pandas==0.23.4 (not sure which version is suitable but pandas is currently not a base functionality)

## Checklist

- [X] Have you ensured the feature or change was not already [reported](https://github.com/NeurodataWithoutBorders/pynwb/issues/) ?
- [X] Have you included a brief and descriptive title?
- [X] Have you included a clear description of the problem you are trying to solve?
- [X] Have you included a minimal code snippet that reproduces the issue you are encountering?

**Comment by ukos-git** [comment]
sorry I just overlooked, that pynwb does not support 3.7, right? - I just tested and there are no problems on 3.6.

Probably closing this would be best, except if the package update is fine.

**Comment by t-b** [comment]
@ukos-git Solved in #652. Please reopen if not.
